pkg/server/metrics_test.go

```diff
@@ -119,6 +113,16 @@ var _ = Describe("Service Metrics", func() {
 	})
 })
 
+func setGockMocks() {
+	gock.New("https://api.github.com").
+		Get("/rate_limit").
+		Reply(200).
+		Body(FromFile("../github/client/test_fixtures/gh/low_rate_limit.json"))
+	gock.New("http://127.0.0.1").
+		Post("").
+		EnableNetworking()
```

Comment:
Don't you think we need to `disableNetworking` after each tests or entire suite?

Reply:
from the gock's code:
```go
// EnableNetworking enables the use real networking for the current mock.
func (r *Request) EnableNetworking() *Request {
	if r.Response != nil {
		r.Response.UseNetwork = true
	}
	return r
}
```
so I guess that it is tied to the particular request. Then calling the `defer gock.OffAll()` should be enough